packages/@aws-cdk/aws-ivs/README.md

```diff
@@ -10,11 +9,76 @@
 > [CFN Resources]: https://docs.aws.amazon.com/cdk/latest/guide/constructs.html#constructs_lib
 
 ---
-
 <!--END STABILITY BANNER-->
 
+Amazon Interactive Video Service (Amazon IVS) is a managed live streaming
+solution that is quick and easy to set up, and ideal for creating interactive
+video experiences. Send your live streams to Amazon IVS using streaming software
+and the service does everything you need to make low-latency live video
+available to any viewer around the world, letting you focus on building
+interactive experiences alongside the live video. You can easily customize and
+enhance the audience experience through the Amazon IVS player SDK and timed
+metadata APIs, allowing you to build a more valuable relationship with your
+viewers on your own websites and applications.
+
 This module is part of the [AWS Cloud Development Kit](https://github.com/aws/aws-cdk) project.
 
+### Channels
+
+An Amazon IVS channel stores configuration information related to your live
+stream. You first create a channel and then contribute video to it using the
+channel’s stream key to start your live stream.
+
+You can create a channel
+
+```ts
+const myChannel = new ivs.Channel(this, 'Channel');
+```
+
+#### Importing an existing channel
+You can reference an existing channel, for example, if you need to create a
```

Comment:
nit: add a blank line after every header - probably something we should enforce with markdown linting in the future.